airflow_dbt_python/utils/version.py

```diff
@@ -0,0 +1,30 @@
+"""Provides utilities to differentiate between installed dbt versions.
+
+These are only used to ensure backwards compatibility with older versions of dbt.
+"""
```

Comment:
I'm afraid this will get too complicated to maintain (it is already). Should we consider not guaranteeing/dropping support for any dbt version older than the last X? X can be set to 3 and this way we could drop all the 1.5 checks.

Is there any reason to keep supporting dbt 1.5?

Reply:
Ending support for older versions of airflow and dbt seems reasonable to me.

I don't know of any real reason to stay on dbt 1.4 or 1.5 and not upgrade to 1.6, except for possible unresolvable dependency conflicts that some users might have. But in any case, I think we should release this (at least one) version with dbt 1.5 support.

For the next release, the decision has to be made, either to drop support for some versions or to follow the [dbt-core eol schedule](https://docs.getdbt.com/docs/dbt-versions/core#latest-releases).

Reply:
Fine with releasing this with dbt 1.5 support.

For the next release I would personally only support the current active version + 1 old version, to ease the maintenance load. But we can discuss that later.